BEIMA.Backend.FT/UserFT.cs

```diff
@@ -188,5 +189,158 @@ public async Task UserInDatabase_DeleteUser_UserDeletedSuccessfully()
             var ex = Assert.ThrowsAsync<BeimaException>(async () => await TestClient.GetUser(userId));
             Assert.That(ex?.StatusCode, Is.EqualTo(HttpStatusCode.NotFound));
         }
+
+        [Test]
+        public async Task UserInDatabase_UpdateUserWithPasswordAndTryLogin_ReturnsUpdatedUserAndSuccessfulLogin()
+        {
+            // ARRANGE
+            var origUser = new User
+            {
+                Username = "user.name",
+                Password = "Abcdefg12345!",
+                FirstName = "Alex",
+                LastName = "Smith",
+                Role = "user"
+            };
+
+            var userId = await TestClient.AddUser(origUser);
+            var origItem = await TestClient.GetUser(userId);
+            Assume.That(origItem, Is.Not.Null);
+
+            var updateItem = new User
+            {
+                Id = userId,
+                Username = "user.name",
+                Password = "NewPassword123!",
+                FirstName = "Alexis",
+                LastName = "Smith",
+                Role = "user"
+            };
+
+            // ACT
+            var updatedUser = await TestClient.UpdateUser(updateItem);
+            
+            // Verify the password has been updated successfully.
+            var loginRequest = new LoginRequest
```

Comment:
We should also do a login before the update and check that the tokens are different.

Reply:
Tokens will always be different on every login attempt, even when using the same exact username and password.

Reply:
Ah, nvm then

Reply:
That way we make sure that the login credentials have changed.

Reply:
Good idea, this has been fixed.